client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -74,6 +76,9 @@ class SiteTools extends Component {
 			"Remove all posts, pages, and media to start fresh while keeping your site's address."
 		);
 
+		const headerCode = translate( 'Header code' );
+		const headerCodeText = translate( 'Insert code within the Header element of your site.' );
```

Comment:
Did you mean `head` element?

Reply:
See above

Reply:
I think this is confusing, and I'd be confused if I were a user trying to add code to my `head`. "Header element" [is something different](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) and I'm convinced we don't mean that.

Reply:
Oops, sorry - that's my bad. I thought I had already updated that.